front/lib/resources/labs_transcripts_resource.ts

```diff
@@ -77,6 +78,31 @@ export class LabsTranscriptsConfigurationResource extends BaseResource<LabsTrans
       : null;
   }
 
+  static async findByWorkspace({
```

Comment:
Does not need to be addressed explicitly in this PR:

I find the current transcripts code unclear in terms of which behaviors are supported. As far as I understand, if a GDrive transcript connection is set up before a Gong connection, it will function correctly, but not vice versa. Would it be feasible to update this method to accept a provider argument? This change would make the behavior more predictable and ensure that both scenarios work as expected.

Reply:
Sounds good, will make another short PR